README.md

```diff
@@ -30,9 +30,12 @@ Have any questions? Check out our [documentation site](https://files.community/d
 
 ### 1: Prerequisites
 
+- Windows 11
```

Comment:
Windows 10 is also supported

Reply:
You are right, i have removed this point